### DISCUSSION TITLE
My account is suspended , i have raised the ticket but no reply for more than 40 days

### DISCUSSION THREAD

**Discussion by aryabasu21** [community/community#144681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My account with username aryabasu17 is being suspended , no official mail received . I have some important things in it , its getting very difficult.

**Comment by BancoCentraI** [comment]
> ### Selecione a área do tópico
> Em geral
> 
> ### Corpo
> Minha conta com o nome de usuário aryabasu17 está sendo suspensa, nenhum e-mail oficial recebido. Tenho algumas coisas importantes nela, está ficando muito difícil.

Sinto muito por saber que sua conta foi suspensa sem um aviso oficial. Quando isso acontece, pode ser realmente frustrante, especialmente se você tem dados importantes ou tarefas em andamento. Aqui estão algumas dicas para tentar resolver o problema:

Verifique sua Caixa de Entrada e Spam: Embora você não tenha recebido um e-mail oficial ainda, é importante verificar as pastas de spam ou lixo eletrônico. Às vezes, a comunicação sobre suspensões ou problemas com a conta pode acabar nessas pastas.

Reveja as Políticas da Plataforma: Se você puder acessar a plataforma de algum outro dispositivo ou conta, veja se há informações sobre as regras ou políticas que possam ter causado a suspensão (como violações de termos de serviço, comportamento suspeito, etc.).

Recuperação de Conta: Muitas plataformas oferecem um processo de recuperação de conta, onde você pode preencher um formulário de contestação ou fornecer detalhes para provar que você é o proprietário da conta. Tente procurar por isso na página de ajuda ou na seção de suporte.

Entre em Contato com o Suporte: Se não houver um e-mail ou aviso claro, tente entrar em contato diretamente com o suporte ao cliente da plataforma. Explique sua situação e forneça o nome de usuário e outros detalhes relevantes para que possam investigar o motivo da suspensão. Certifique-se de incluir informações como o seu nome completo, e-mail registrado e quaisquer mensagens de erro que você tenha visto.

Aguarde Resposta: Após entrar em contato com o suporte, é importante aguardar a resposta. As plataformas normalmente têm um tempo de resposta padrão, mas dependendo da gravidade da situação, o processo pode levar mais tempo.

Se você me disser em qual plataforma sua conta foi suspensa, posso tentar fornecer orientações mais específicas sobre como proceder para resolver esse problema.

**Reply by aryabasu21** [reply]
It's my GitHub account

**Reply by BancoCentraI** [reply]
Entendo sua frustração com essa situação. Infelizmente, pode ser bastante demorado obter uma resposta direta do suporte do GitHub, especialmente para casos mais complicados. Vou compartilhar algumas orientações que podem ajudar a tentar acelerar o processo:

1. Verifique o status do seu ticket
Às vezes, você pode revisar o status do ticket no email que você utilizou para abrir a solicitação. Verifique se o GitHub enviou alguma atualização ou uma solicitação de informações adicionais.
Se você não recebeu nada, tente buscar no seu filtro de spam/lixo eletrônico, pois alguns emails do suporte podem acabar lá.
2. Aperte o Follow-Up
Responda diretamente ao ticket original (caso você tenha um número de referência) pedindo uma atualização. Tente ser educado e direto.
Você pode enviar um follow-up email para support@github.com com o número do ticket original, reforçando que já se passaram mais de 40 dias e você precisa de uma atualização.
3. Contate o Suporte pelo Twitter (X)
O GitHub é bastante ativo no Twitter (agora X). Tente enviar uma mensagem mencionando o perfil deles: [@GitHubSupport](https://twitter.com/GitHubSupport). Algumas vezes, isso pode acelerar o processo, já que eles preferem resolver problemas publicamente para manter a reputação.
Use uma mensagem curta, informando que abriu um ticket há 40 dias e não teve resposta. Inclua o número do ticket (sem expor muitas informações pessoais).
4. Outras Alternativas
GitHub Community Forum: Às vezes, moderadores ou funcionários do GitHub presentes no fórum podem ajudar a acelerar o processo. Confira: [GitHub Community Forum](https://github.community/).
Verifique as Diretrizes: Garanta que você não violou nenhuma das [Diretrizes da Comunidade do GitHub](https://docs.github.com/en/site-policy). Isso pode dar uma ideia do motivo da suspensão, caso ainda não tenha sido informado.
5. Considere Abrir Outro Ticket
Se realmente não houver resposta, você pode tentar abrir um novo ticket explicando o problema e referenciando o número do ticket original. No entanto, faça isso como último recurso, para evitar que sua solicitação seja interpretada como spam.
Observações Finais
O GitHub é uma plataforma muito usada, e o volume de solicitações que o suporte recebe é enorme. Infelizmente, isso pode resultar em tempos de resposta longos, especialmente para contas gratuitas. Se você tiver uma conta Pro ou Team, há maior prioridade para o suporte.
